app/views/service/_svcs_show.html.haml

```diff
@@ -28,7 +28,7 @@
             .form-horizontal.static
               .form-group
                 .col-md-9
-                  - htm = @job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'")
+                  - htm = @job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'").gsub(/{{/, '\{{').gsub(/}}/, '\}}')
```

Comment:
@h-kataria - We'll need this `.gsub(/{{/, '\{\{').gsub(/}}/, '\}\}')` (and on line 41) for it to work.